test/community.js

```diff
@@ -0,0 +1,147 @@
+    expect(community.name).to.equal(communityCustomName || communityName)
+    expect(community.wallets).to.be.an('array')
+    expect(community.wallets).to.have.lengthOf(3)
+    expect(community.mnemonic).to.be.a('string')
+    expect(community.defaultCurrency.toString()).to.equal(currency.id)
+    community.wallets.forEach(wallet => {
+      expect(['manager', 'users', 'merchants']).to.contain(wallet.type)
+      expect(wallet.address).to.be.a('string')
+      expect(wallet.index).to.be.a('number')
+      expect(wallet.balances).to.be.an('array')
+      expect(wallet.balances).to.have.lengthOf(1)
+      expect(wallet.balances[0].currency.toString()).to.equal(currency.id)
+      expect(wallet.balances[0].blockchainBalance.toNumber()).to.equal(0)
+      expect(wallet.balances[0].offchainBalance.toNumber()).to.equal(0)
+    })
+  }
+
+  before(async function () {
+    // console.log('--- before [community] ---')
+    this.timeout(60000)
+
+    const mmLib = await EllipseMarketMakerLib.new()
+
+    cln = await ColuLocalNetwork.new(CLN_MAX_TOKENS)
+    await cln.makeTokensTransferable()
+
+    const currencyFactory = await CurrencyFactory.new(mmLib.address, cln.address, {from: accounts[0]})
+    const result = await currencyFactory.createCurrency('TestLocalCurrency', 'TLC', 18, CC_MAX_TOKENS, 'ipfs://hash', {from: accounts[0]})
+    ccAddress = result.logs[0].args.token
+    const cc = await ColuLocalCurrency.at(ccAddress)
+
+    mmAddress = await currencyFactory.getMarketMakerAddressFromToken(ccAddress)
+    const mm = await EllipseMarketMaker.at(mmAddress)
+
+    await currencyFactory.openMarket(ccAddress)
+
+    osseus = await Osseus.get()
+    osseus.cwd = osseus.cwd || cwd
+    osseus.db_models = osseus.db_models || {
+      currency: require(path.join(cwd, 'modules/db/models/currency'))(osseus.mongo),
+      community: require(path.join(cwd, 'modules/db/models/community'))(osseus.mongo)
+    }
+    osseus.lib = osseus.lib || {
+      Currency: require(path.join(cwd, 'modules/lib/models/currency'))(osseus),
+      Community: require(path.join(cwd, 'modules/lib/models/community'))(osseus)
+    }
+  })
+
+  beforeEach(function () {
+    // console.log('--- beforeEach [community] ---')
+    Object.keys(osseus.db_models).forEach(model => {
+      // console.log(`model: ${model}`)
+      osseus.db_models[model].getModel().remove({}, () => {})
+    })
+  })
+
+  it('should create a community', async () => {
+    let currency = await osseus.lib.Currency.create(ccAddress, mmAddress, ccABI, mmABI)
+    let community = await osseus.lib.Community.create(communityName, currency.id)
+    validateCommunity(community, currency)
+  })
+
+  it('should get community (by id)', async () => {
+    let currency = await osseus.lib.Currency.create(ccAddress, mmAddress, ccABI, mmABI)
+    let community = await osseus.lib.Community.create(communityName, currency.id)
+    community = await osseus.db_models.community.getById(community.id)
```

Comment:
can you check the retrieved community against the created community, instead of override the variable and just check that it is a community?